src/core/midiPorts.cpp

```diff
@@ -468,4 +468,28 @@ void midiReceive(const MidiMsg& mm, const std::string& recipient)
 }
 
 
+/* -------------------------------------------------------------------------- */
+
+std::string shortName(const std::string& port) {
+
+	std::string o = port;
+
+	// Remove non-alphanumeric chars
+	o.erase(std::remove_if(o.begin(), o.end(), 
+	[] (auto const& c) -> bool {return !std::isalnum(c);}), o.end());
+
+	// "y" is a vowel in most languages, let's just keep it, aight?
+	std::string vowels = "aeiouyAEIOUY";
+
+	o.erase(std::remove_if(o.begin(), o.end(), 
+	[&] (auto const& c) -> 
+	bool {return (vowels.find(c)!=std::string::npos);}), o.end());
```

Comment:
Why not fusing the 2 erase just by providing a more fused single lambda?

Reply:
Could be done, but I find this approach easier to read.